### DISCUSSION TITLE
Wrong TSML listings showing up in Meeting Guide: District website not in sync with the larger Area web presence

### DISCUSSION THREAD

**Discussion by benlambm** [code4recovery/12-step-meeting-list#1361] [Q&A] [answered]
Our local District22 maintains its own website and updates to local meetings using TSML at harrisonburgaa.org going on five years now. Occasionally a conflict arises between newer, correct Meeting information and older, incorrect info on the Area website (aavirginia.org which also uses TSML). 

Anyway, in the past TSML would prefer local over larger regional, I think, but somehow this got broke again and it shows up The Meeting Guide app with one of our local meetings listing the Area's version instead of the source of truth. 

Again, the problem is that The Meeting Guide app is showing the incorrect details from the area website and not listing our correct version at all. 

PS - Our local meetings/groups submit Meeting updates/changes to our District webmaster, currently me, to post to harrisonburgaa.org and things seems to be hopelessly always out of sync with the larger Area web presence...hope that makes sense

**Comment by joshreisner** [comment, marked as answer]
i'm sorry to hear that the area site is out of date. perhaps they could start importing data from your district site?

at any rate, you can let the General Service Office know the specific listings and they can switch their importer to import from your site instead of from the Area. perhaps they know why the listings changed. we don't have that information
